gatsby-node.js

```diff
@@ -19,24 +19,24 @@ exports.createPages = ({ graphql, boundActionCreators }) => {
               }
             }
           }
-          `
+        `
       ).then(result => {
         if (result.errors) {
           console.log(result.errors)
           reject(result.errors)
         }
 
         const posts = result.data.allContentfulBlogPost.edges
-        posts.forEach((post, index) => {
+        posts.forEach(post => {
           createPage({
             path: `/blog/${post.node.slug}/`,
             component: blogPost,
             context: {
-              slug: post.node.slug
+              slug: post.node.slug,
```

Comment:
do you have a linter that adds trailing commas? (see also `gatsby-config.js` above)

Reply:
I did run userland's prettier configuration but didn't introduced it to keep the changes to a minimum. I did request for trailing commas just because I used the prettier config from userland.